src/lib.rs

```diff
@@ -234,7 +234,7 @@ pub mod pallet {
 		/// Disputes a channel in case of a dishonest participant.
 		///
 		/// Can only be called with a non-finalized state that is signed by
-		/// all participants.
+		/// all participants when there is currently no on-going dispute.
```

Comment:
```suggestion
		/// all participants if there is no on-going dispute.
```

Reply:
I just saw that this is actually rebased onto #3, fixed it there instead.